Anamnesis/Languages/en.json

```diff
@@ -310,6 +310,7 @@
 	"Pose_WarningNotFrozen": "Actor motion must be disabled in the Group Pose settings, or Freeze Positions must be enabled in the pose options.",
 	"Pose_WarningQuit": "Are you sure you want to quit? Current pose will be lost.",
 	"Pose_WarningExpresionPositions": "You are attempting to load an expression while Position [P] mode is active; this may have undesired results. Continue?",
+	"Pose_WarningExpresionOld": "You are attempting to load an expression from a pose file that was saved prior to DT; this will have undesired results. Continue?",
```

Comment:
Reword this to something that implies "It appears you are". Just for vanity in case people save without face bones. 
Alternatively if you want to make it a bit more fail save, you can consider implementing a check if face bones are present at all, that way figuring out if JUST this bone is missing or all face bones.

Reply:
Good call! Luckily face bones are prefaced with j_f_, so implementing a check was easy.